### ISSUE TITLE
xml error 4 not well-formed (invalid token)

### ISSUE THREAD

**Issue by zl1234aaa** [mariotaku/moonlight-tv#447]
### Moonlight Version

1.6.18

### Device Type

webOS TV

### Device Model

lg c9

### Device OS

webos 4.10.0-17（goldilocks-grampians）

### Host OS

Windows

### Host Software

Sunshine

### Host Setup

i7-11700K gtx2080 16g ram with win11 23h2

### Bug Description

Hello i have installed the sunshine app, it's running OK.There is no problem when i use the phone to connect pc.I have installed zerotierone.When i place my ip on my LG TV moonlight client it gives me the return "xml error 4 not well-formed (invalid token)", if i tried my public internet ip it gives me timeout, my ethernet is private, firewall is disable, windows is configured with all ports, i don't know what i can do, can anybody help ?

**Comment by andregamma** [comment]
Can confirm this also happens in LG C4, moonlight version `1.6.19`. I can connect without any problems with my phone, but I'm unable to connect with the moonlight app on TV.

**Comment by mariotaku** [comment]
@andregamma Did you type the IP manually? I'm unable to reproduce, can you tell me all the steps you see the issue?

**Comment by andregamma** [comment]
> @andregamma Did you type the IP manually? I'm unable to reproduce, can you tell me all the steps you see the issue?

The steps are nothing unusual... that's why it's a strange behavior.

1. Start Sunshine
2. Open the Moonlight TV app
3. Type in the IP manually

I have to type it in manually because the connection doesn't show up on the home screen.

**Comment by mariotaku** [comment]
@andregamma If the TV and the PC are in the same network, auto discovery should work though... Are they in the same network?

**Comment by andregamma** [comment]
Yes, both same network. I tested switching between 5ghz and 2.4ghz wifi network, same error on both. My last ideia was to try using an ethernet cable and for my surprise it also dont work.

Maybe it's a regional thing on the TV that affects moonlight or an ISP/Router setting? If it's the last, I'm afraid I can do nothing about it :/

**Model:**
OLEO55C4PSA.AWZQLJZ

**WebOS Version:**
9.1.0-48

**Comment by andregamma** [comment]
According to Sunshine logs, it appears that the error that makes me unable to connect is the following:
```
Warning: SSL Verification error :: self-signed certificate
```

@mariotaku considering that I'm able to use Moonlight Android without any problems, this has to be a Moonlight TV issue, right? Do you have any ideas?
